dashboard/db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema.define(version: 2023_06_09_220913) do
 
-  create_table "activities", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
```

Comment:
Should these changes be in this PR?

Reply:
Nope sorry that was an accident when trying to merge in staging, I'm fixing that currently!